### PR TITLE
EDU-13931: Update navigation.json for new release note

### PR DESCRIPTION
**Pull request by mariana-caetano**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/components/last-updates-section/index.tsx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import { getMessages } from 'utils/get-messages'
...
 
 const lastReleaseNote: CardProps = {
-  action: {
+   action: {
     type: 'improved',
     description:
-      'Store Framework: Simplified custom header management now available in VTEX admin',
-    date: new Date('12/05/2024'),
+      'FastStore: `faststore.config.js` file is now `discovery.config.js`',
+    date: new Date('17/12/2024'),
```
Comment (carolinamenezes) [inline src/components/last-updates-section/index.tsx:13]: Ajustar data

File: public/navigation.json [hunk 2]
```diff
@@ -14852,6 +14852,13 @@
               "origin": "",
               "type": "category",
               "children": [
+                {
+                  "name": "FastStore: `faststore.config.js` file is now `discovery.config.js`",
+                  "slug": "2024-12-13-discovery-config-file",
```
Comment (mariana-caetano) [inline public/navigation.json:14857]:
```suggestion
                  "slug": "2024-12-16-discovery-config-file",
```

File: src/components/last-updates-section/index.tsx (hunk 1 above)
Comment (mariana-caetano) [inline src/components/last-updates-section/index.tsx:13]:
```suggestion
    date: new Date('16/12/2024'),
```

File: src/components/last-updates-section/index.tsx [hunk 3]
```diff
@@ -6,11 +6,11 @@ import { getMessages } from 'utils/get-messages'
 import styles from './styles'
 
 const lastReleaseNote: CardProps = {
-  action: {
+   action: {
     type: 'improved',
     description:
-      'Store Framework: Simplified custom header management now available in VTEX admin',
-    date: new Date('12/05/2024'),
+      'FastStore: `faststore.config.js` file is now `discovery.config.js`',
```
Comment (mariana-caetano) [inline src/components/last-updates-section/index.tsx:12]:
```suggestion
      'FastStore: faststore.config.js file is now discovery.config.js',
```

File: src/components/last-updates-section/index.tsx [hunk 4]
```diff
@@ -6,11 +6,11 @@ import { getMessages } from 'utils/get-messages'
 import styles from './styles'
 
 const lastReleaseNote: CardProps = {
-  action: {
+   action: {
```
Comment (mariana-caetano) [inline src/components/last-updates-section/index.tsx:9]:
```suggestion
  action: {
```

File: src/components/last-updates-section/index.tsx (hunk 1 above)
Comment (mariana-caetano) [reply src/components/last-updates-section/index.tsx]: Ajustado!

**Pull request merged**